### ISSUE TITLE
**FEATURE REQUEST** -- Regular calls

### ISSUE THREAD

**Issue by udni** [WhisperSystems/RedPhone#413]
The ability to perform regular calls through RedPhone -- just like you can send insecure text messages in TexSecure. The reason I woud like this, is because I prefer using one app for everything -- just as I do with TextSecure.

**Comment by Thunderpig72** [comment]
Yup, I agree. If that's not possible then RedPhone/Signal could make an encrypted call, and then when there's no response it automatically makes a regular call.

**Comment by kelna** [comment]
I'm not sure this would be so good. I like to always know if my communication is secure or not. It is very important, and if RedPhone automatically falls back to unencrypted, than you always have to double check. I wouldn't like that. In fact, I don't like it very much that encrypted and unencrypted messages are not separated very well in TextSecure.

**Comment by gabsoftware** [comment]
I think a good solution would be to ask the caller if he wants to fall back to insecure call in case the recipient does not have RedPhone. Then if the caller accept, the call is forwarded to insecure phone app.

**Comment by kelna** [comment]
Exactly, so it should be opt-in, not opt-out. :smile:
